### PR TITLE
Add Swedish translation

### PR DESCRIPTION
**Pull request by ming-00**

### CODE REVIEW

File: l10n/bundle.l10n.se.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	"gray": "Grå",
+	"purple": "Lila",
+	"red": "Röd",
+	"white": "Vit",
+	"cat": "Kat",
+	"clippy": "Clippy",
+	"cockatiel": "Kakadua",
+	"crab": "Krabba",
+	"dog": "Hund",
+	"mod": "Mod (the .NET bot)",
```
Comment (tonybaloney) [inline l10n/bundle.l10n.se.json:26]: Does `the` and `bot` have a translated equivalent?

**Comment by tonybaloney** [comment]
Thanks for the contribution!

**Comment by tonybaloney** [comment]
Also, I think the 2 letter language code is `sv` not `se`.

**Comment by ming-00** [comment]
Updated! Thank you :-)

**Review by tonybaloney — APPROVED** [review]

**Pull request merged**